nx-dev/nx-dev/pages/[...segments].tsx

```diff
@@ -44,13 +45,20 @@ export function DocumentationPage({
   const { value: storedFlavor, setValue: setStoredFlavor } = useStorage(
     'flavor'
   );
+  const { setValue: setStoredVersion } = useStorage('version');
   const [dialogOpen, setDialogOpen] = useState(false);
+  const [navIsOpen, setNavIsOpen] = useState(false);
 
   useEffect(() => {
     if (!isFallback) {
       setStoredFlavor(flavor.value);
     }
   }, [flavor, isFallback, setStoredFlavor]);
+  useEffect(() => {
```

Comment:
I'm thinking we can can abstract this so all the key-value pairs can be synced up with one hook, but we can do  that clean up later since it's not that important.